src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/IWantDocumentRule.java

```diff
@@ -22,8 +22,7 @@ public boolean processAddIWantItemRules(IWantDocument document, IWantItem item,
         boolean valid = true;
         if (StringUtils.isBlank(item.getItemDescription())) {
             valid = false;
-            String attributeLabel = getDataDictionaryService().getDataDictionary().getBusinessObjectEntry(
-                    IWantItem.class.getName()).getAttributeDefinition(CUPurapPropertyConstants.IWNT_ITEM_DESC).getLabel();
+            String attributeLabel = getDocumentDictionaryService().getDocumentEntry(IWantItem.class.getName()).getAttributeDefinition(CUPurapPropertyConstants.IWNT_ITEM_DESC).getLabel();
```

Comment:
Should this call use dataDictionaryService or businessObjectDictionaryService instead? IWantItem is a business object, not a document.

Reply:
updated